website/src/content/development/forms.mdx

```diff
@@ -0,0 +1,3 @@
+---
+title: Forms
```

Comment:
Yes, we definitely want to document this. However, since we didn't have documentation for forms before, we decided to not include it in the release.

In other words, it's not a blocker for the release. But we will document this for sure.